includes/application/AcceptAppl.php

```diff
@@ -125,7 +125,7 @@
 			}
 			else
 			{
-				$days = ( strtotime($enD) - strtotime($stD) )/ (60*60*24);
+				$days = ( strtotime($enD) - strtotime($stD) )/ (60*60*24) + 1;
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->